README.md

```diff
@@ -1,193 +1,92 @@
-[Zowe](https://www.zowe.org/) is a project hosted by the [Open Mainframe Project](https://www.openmainframeproject.org/), a [Linux Foundation](https://www.linuxfoundation.org/) project.
+[Zowe Explorer](https://github.com/zowe/community#zowe-explorer) is a sub-project of Zowe, focusing on modernizing mainframe experience. [Zowe](https://www.zowe.org/) is a project hosted by the [Open Mainframe Project](https://www.openmainframeproject.org/), a [Linux Foundation](https://www.linuxfoundation.org/) project.
 
-The Zowe Explorer extension modernizes the way developers and system administrators interact with z/OS mainframes. Working with data sets and USS files from VSC can be more convenient than using 3270 emulators, and complements your Zowe CLI experience. The extension provides the following benefits:
+The Zowe Explorer extension modernizes the way developers and system administrators interact with z/OS mainframes by:
 
-* Enables you to create, modify, rename, copy, and upload data sets directly to a z/OS mainframe.
-* Enables you to create, modify, rename, and upload USS files directly to a z/OS mainframe.
-* Provides a more streamlined way to access data sets, uss files and jobs.
-* Lets you create, edit, and delete Zowe CLI `zosmf` compatible profiles.
-* Lets you use the Secure Credential Store plug-in to store your credentials securely in the settings.
+* Enabling you to create, modify, rename, copy, and upload data sets directly to a z/OS mainframe.
+* Enabling you to create, modify, rename, and upload USS files directly to a z/OS mainframe.
+* Providing a more streamlined way to access data sets, uss files and jobs.
+* Letting you create, edit, and delete Zowe CLI `zosmf` compatible profiles.
+* Letting you use the Secure Credential Store plug-in to store your credentials securely in the settings.
 
-**Note:** The Zowe Explorer is powered by [Zowe CLI](https://zowe.org/home/). The extension demonstrates the potential for plug-ins powered by Zowe.
-
-**Tips:**
-
-* For information about how to install the extension from a `VSIX` file and run system tests on the extension, see the [Developer README](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README.md).
-* For information about how to develop for Eclipse Theia, see [Theia README](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README-Theia.md).
-* For information about how to create a VSCode extension for Zowe Explorer (a Zowe Explorer extension) see [VSCode extensions for Zowe Explorer](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README-Extending.md).
-* For questions and general guidance, please visit #zowe-explorer channel on [Slack](https://openmainframeproject.slack.com/)
+Join the **#zowe-explorer** channel on [Slack](https://openmainframeproject.slack.com/) to stay in touch with the Zowe community.
```

Comment:
At this point, do you want to provide a link to the Zowe Docs pages? "Visit the Zowe Explorer section of the Zowe Documentation for comprehensive information about the extension" or something like that. I just want to make sure that it is very early in the readme.

Reply:
Absolutely. Putting it above the Slack channel reference.